src/collections/maple_tree/mod.rs

```diff
@@ -1198,110 +1482,149 @@ mod tests {
         assert!(rn.move_pivots_left(0, 1) == false);
 
         assert!(rn.move_pivots_left(1, 1) == true);
-        check_node_state([17, 19, 0, 0, 0, 0, 0],[ 2, 0, 0, 0, 0, 0, 0, 0], &rn);
+        check_node_state([17, 19, 0, 0, 0, 0, 0], [2, 0, 0, 0, 0, 0, 0, 0], &rn);
```

Comment:
You could make a "populate node state" similar to check, with the key/value pairs in a vec to make some of these fixtures shorter?

Reply:
good idea, I will start on this today